src/node/mod.rs

```diff
@@ -4,9 +4,9 @@ pub mod constants;
 
 use std::error::Error;
 use std::fmt::{self, Debug, Display, Formatter};
-use std::io;
 use std::os::unix::io::AsFd;
 use std::path::{Path, PathBuf};
+use std::{fs, io};
```

Comment:
We should probably merge all module imports, or move `fmt::{self` here and flatten that a bit.

Reply:
I moved `fmt` down to this import line. Please take another look.

Reply:
I think I should have been more elaborate here (but wasn't, since it's not related to this PR): personally I don't like importing _both_ a module and _some_ of its items. In this case `fmt` is only used in the `impl` blocks for `fmt::Result`, but `Debug`/`Display`/`Formatter` are globally imported.

If that's to disambiguate the `Result` from the prelude, my personal preference is to import only `fmt` and qualify `fmt::Debug`/`fmt::Display`/`fmt::Formatter` :)